02_tidyverse.qmd

```diff
@@ -17,13 +18,23 @@ library(tidyverse)
 theme_set(theme_minimal())
 ```
 
-# Review
+
+
+```{r}
+#| echo: false
+
+exercise_number <- 1
+
+```
+
+
+## Review
```

Comment:
I wonder if there is an elegant way to style the review sections so they stand out as different. Maybe we can add an CSS class?

Reply:
I'm not going to work on this right now and add an issue and assign me. I think one thing we could think about is using the [tip callout](https://quarto.org/docs/authoring/callouts.html). 

Also happy to look into that custom CSS class vision, but I just haven't seen this before and could use a bit more guidance on how this could work.

Reply:
Reasonable!